environments/sandbox/global.tfvars

```
@@ -0,0 +1,39 @@
+project   = "hmcts"
+component = "frontdoor"
+location  = "uksouth"
+env       = "sandbox"
+waf_mode  = "Detection"
+
+custom_domain_name = "platform.hmcts.net"
+
+backend_domain = [
+  "uksouth.platform.hmcts.net",
+  "ukwest.platform.hmcts.net",
+]
+
+# Sample data
+frontend_with_disabled_waf_rules = [
+  {
+    name = "cmc"
+    rules = {
```

this should probably be:
```suggestion
    disabled_rules = {
```